### ISSUE TITLE
Discussion on License choice

### ISSUE THREAD

**Issue by franklu2014** [UBC-MDS/rimgtool#62]
Hi team, let's use this place to have some discussion on license.

**Comment by rita-ni** [comment]
I found this resource to be very helpful : https://choosealicense.com/

Since we want it to be simple and permissive, shall we go with the MIT license?

> The MIT License is short and to the point. It lets people do almost anything they want with your project, like making and distributing closed source versions.

Above is the description for MIT licensing from the website

**Comment by franklu2014** [comment]
yeah, the MIT license looks suitable for our project.  Since we didn't really create new algorithms or processes, we want people to use it for free.

**Comment by Margaret8521** [comment]
We also want others to contribute to our project, so I think the MIT license is good enough.

**Comment by franklu2014** [comment]
Yeah, we do use free images from the Internet and want to get some involvement from interested people.  I think MIT license is the one.

**Comment by rita-ni** [comment]
> Yeah, we do use free images from the Internet and want to get some involvement from interested people. I think MIT license is the one.

I agree with your point, we would use MIT for this project then
